DevCycle.SDK.Server.Cloud/Api/DVCCloudClient.cs

```diff
@@ -86,6 +86,8 @@ public async Task<Variable<T>> VariableAsync<T>(User user, string key, T default
             try
             {
                 variable = await GetResponseAsync<Variable<T>>(user, urlFragment, queryParams);
+                variable.DefaultValue = defaultValue;
+                variable.IsDefaulted = false;
```

Comment:
How do you know whether or not it was defaulted?

Reply:
cuz if its defaulted then `GetResponseAsync` will throw (because the request is 404)